tests/riotboot_flashwrite/README.md

```diff
@@ -8,11 +8,38 @@ over network without *any* kind of authentication or other security!
 Please see the README of examples/nanocoap_server for instructions on how to
 set up a network for testing.
 
-# How to test
+## Requirements
+
+This test uses libcoap to send the firmware to the device over coap.
```

Comment:
According to the test script, it's rather using aiocoap Python package and not libcoap:

```suggestion
This test uses [aiocoap](https://pypi.org/project/aiocoap/) to send the firmware to the device over coap.
```